lib/app/dataMixin.js

```diff
@@ -28,6 +28,36 @@ export default {
     $site () {
       return store.siteData
     },
+    $title () {
+      const title = this.$site.title
+      return typeof title === 'object'
+        ? title[this.$basepath]
+        : title
+    },
+    $description () {
+      const description = this.$site.description
+      return typeof description === 'object'
+        ? description[this.$basepath]
+        : description
+    },
+    $langConfig () {
+      const { langs } = this.$site
+      let defaultLang
+      const lang = (langs || []).find((lang, index) => {
```

Comment:
Buble does not polyfill `Array.prototype.find` so we have to avoid using it in order to make the site run in IE11.

Reply:
Thanks! all incompatible API issues are fixed at 1c31f821ef9ea079cd8ab40586e2c106f28f7a38